### DISCUSSION TITLE
fprime-util new support on fprime-tools 3.1.1

### DISCUSSION THREAD

**Discussion by ebenezerarunkumar** [nasa/fprime#1669] [Q&A]
Hi!
I'm on fprime-tools v3.1.1 and I'm trying to use `fprime-util new`, but I keep getting the following output no matter what arguments I use...

`[ERROR] template() takes 4 positional arguments but 5 were given`

Is anyone else running into this same issue?

**Comment by LeStarch** [comment]
`fprime-util new` isn't currently updated for FPP models.  It is on our list.

**Reply by ThibFrgsGmz** [reply]
Side note, the ticket is here: https://github.com/fprime-community/fprime-tools/issues/37

**Reply by SterlingPeet** [reply]
Hmm, seems like it should error, rather than crash with an unhandled exception.
